src/Symfony/Bridge/Doctrine/Tests/ContainerAwareEventManagerTest.php

```diff
@@ -245,16 +245,14 @@ public function testGetListenersForEventWhenSubscribersArePresent()
         $this->assertSame([$subscriber1, $listener1, $listener2], array_values($this->evm->getListeners('foo')));
     }
 
-    /**
-     * @group legacy
-     */
     public function testGetListeners()
     {
         $this->container->set('lazy', $listener1 = new MyListener());
         $this->evm->addEventListener('foo', 'lazy');
         $this->evm->addEventListener('foo', $listener2 = new MyListener());
 
-        $this->expectDeprecation('Since symfony/doctrine-bridge 6.2: Calling "Symfony\Bridge\Doctrine\ContainerAwareEventManager::getListeners()" without an event name is deprecated. Call "getAllListeners()" instead.');
+        $this->expectExceptionMessage('"Symfony\Bridge\Doctrine\ContainerAwareEventManager::getListeners()" must be called with an event name.');
+        $this->expectException(\InvalidArgumentException::class);
 
         $this->assertSame([$listener1, $listener2], array_values($this->evm->getListeners()['foo']));
```

Comment:
We expect the call to throw, so performing assertions on the return value does not make sense anymore.

Reply:
Oh that's right. Fixed :slightly_smiling_face: